### ISSUE TITLE
Maybe a bug in dataloader

### ISSUE THREAD

**Issue by wangyuxin87** [lcy0604/EraseNet#2]
Thanks for your work, there may be a bug in dataloader.py, where all_input may be all_img in [this line](https://github.com/lcy0604/EraseNet/blob/6a104da43edca7ac53bce72f74803ec7b465f8d2/data/dataloader.py#L61).

**Comment by lcy0604** [comment]
> Thanks for your work, there may be a bug in dataloader.py, where all_input may be all_img in [this line](https://github.com/lcy0604/EraseNet/blob/6a104da43edca7ac53bce72f74803ec7b465f8d2/data/dataloader.py#L61).

Thanks for pointing out this bug. It is fixed now.

**Comment by wangyuxin87** [comment]
The paper claims that you use SCUT-EnsText and SCUT-Syn to train the model. I would like to ask whether you directly integrate the two data for training or firstly train to convergence on SCUT-Syn and then fine-tune on SCUT-EnsText? Looking forward to your reply!

**Comment by lcy0604** [comment]
> The paper claims that you use SCUT-EnsText and SCUT-Syn to train the model. I would like to ask whether you directly integrate the two data for training or firstly train to convergence on SCUT-Syn and then fine-tune on SCUT-EnsText? Looking forward to your reply!

In my experiments, I train EnsText and Syn respectively.

**Comment by wangyuxin87** [comment]
Hi, I find the mask label is not avaliable in the [SCUT-Syn](https://github.com/HCIILAB/Scene-Text-Removal), which is necessary for your work. Can you provide this label?
